src/Client.php

```diff
@@ -172,6 +172,11 @@ private function configureDefaults(array $config)
             $defaults['proxy']['https'] = $proxy;
         }
 
+        if ($noProxy = getenv('NO_PROXY')) {
+            $cleanedNoProxy = preg_replace('/\s/', '', $noProxy);
```

Comment:
Yes, sure, but I don't know if it is the best solution. On these lines, I remove all spaces / tabs of a list of coma separated string. So my first solution allow to remove all spaces on the string. Your just remove spaces on the begining and at the end, not all.